### PR TITLE
GraphQl-662: Test coverage of USPS shipping method

### PR DESCRIPTION
**Pull request by atwixfirster**
https://github.com/magento/graphql-ce/issues/662

### Description (*)
Works with sandbox USPS account.

@naydav , unfortunately I did not find a way how to emulate USPS service without values in the "User ID" and "Password" fields.

On my local instance to verify `Magento\GraphQl\Usps\SetUspsShippingMethodsOnCartTest`::`testSetUpsShippingMethod()` test I used my USPS sandbox account.

Thank you!

### CODE REVIEW

**Comment by atwixfirster** [comment]
@naydav , I've a proposition for you here.

On my local environment I've used my USPS sandbox account.

May be a good idea to have the appropriate one for Magento test instance. To provide USPS credentials and hide them from the 3rd eyes my proposition is use 
`dev/tests/api-functional/config/config-global.php` config file with the next content (for USPS):
```
<?php
/**
 * Copyright © Magento, Inc. All rights reserved.
 * See COPYING.txt for license details.
 */

return [
    'carriers' => [
        'usps' => [
            'userid' => '0:3:xxxxxxxx==',
            'password' => '0:3:yyyyyyyyy==',
        ]
    ]
];
```

where `0:3:xxxxxxxx==` is an encoded `User ID` and `0:3:yyyyyyyyy==` is an encoded account `Password`.

Please let me know your thoughts. 

Thank you!

File: dev/tests/api-functional/testsuite/Magento/GraphQl/Usps/SetUspsShippingMethodsOnCartTest.php [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+{
+    /**
+     * Defines carrier code for "USPS" shipping method
+     */
+    const CARRIER_CODE = 'usps';
+
+    /**
+     * Defines method code for the "Retail Ground" USPS shipping
+     */
+    const CARRIER_METHOD_CODE_GROUND = '4';
```
Comment (naydav) [inline dev/tests/api-functional/testsuite/Magento/GraphQl/Usps/SetUspsShippingMethodsOnCartTest.php]: Need to create `@dataprovider` and cover all of possible metods

**Review by naydav — CHANGES_REQUESTED** [review]

**Comment by atwixfirster** [comment]
@naydav , ticket is ready for your review.

Thank you!

**Review by naydav — CHANGES_REQUESTED** [review]
@atwixfirster 
As we talk early, need to introduce some mock object instead of 'real caller' object
Our tests should not depend on external system

The best way is creating preference in GraphQLTestModule and replace real object on our internal mock object

**Review by lenaorobei — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lenaorobei, thank you for the review.
[ENGCOM-5284](https://jira.corp.magento.com/browse/ENGCOM-5284) has been created to process this Pull Request
:eight_spoked_asterisk: @lenaorobei, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Review by naydav — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @naydav, thank you for the review.
[ENGCOM-5284](https://jira.corp.magento.com/browse/ENGCOM-5284) has been created to process this Pull Request
:eight_spoked_asterisk: @naydav, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Review by naydav — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @naydav, thank you for the review.
[ENGCOM-5284](https://jira.corp.magento.com/browse/ENGCOM-5284) has been created to process this Pull Request

**Pull request merged**